tests/unit_tests/bittensor_tests/test_balance.py

```diff
@@ -318,14 +301,12 @@ def test_balance_rfloordiv_other_not_balance(self, balance: Union[int, float], b
             rao_ = balance
         elif isinstance(balance, float):
             rao_ = int(balance * pow(10, 9))
-        if isinstance(balance2, int):
-            rao2_ = balance2
-        elif isinstance(balance2, float):
-            rao2_ = int(balance2 * pow(10, 9))
+        # assume balance2 is a rao value
+        rao2_ = balance2
 
         quot_ =  balance2_ // balance_ # This is an rfloordiv
         assert isinstance(quot_, Balance)
-        assert CLOSE_IN_VALUE(quot_.rao, 5) == rao2_ // rao_   
+        self.assertAlmostEqual(quot_.rao, rao2_ // rao_, delta=5)
 
```

Comment:
Can you also add some tests for the value errors in balance.py?
i.e. with raises () ....